engine/crates/gateway-v2/src/lib.rs

```diff
@@ -106,6 +109,11 @@ pub struct Session {
 
 impl Session {
     pub async fn execute(self, ctx: &impl RequestContext, request: Request) -> Response {
+        let gql_request_span = grafbase_tracing::span::gql::GqlRequestSpan::new()
+            .with_operation_name(request.operation_name())
```

Comment:
operation name needs to be added later actually, after parsing the request. I'll fix that later, I'll rework some of the gateway code anyway